### PR TITLE
Separate storage cost from message cost in billing transactions

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review, though I didn't try grok much of those insane sql queries.

**Comment by justinvdm** [comment]
Ah, looks like the forms still need to change, not actually ready for review yet.

**Comment by justinvdm** [comment]
Should be ready for review now.

**Comment by justinvdm** [comment]
Just merged in develop. Wondering if the recently added migration for the new `TransactionArchive` model is going to complicate things, since this PR also adds a migration.

**Comment by hodgestar** [comment]
If they're orthogonal you can just rename your migration file to have a
later number. Otherwise just delete your migration and re-create it?

**Comment by justinvdm** [comment]
Ok thanks, will do second one.

**Comment by justinvdm** [comment]
Forgot to add storage credits when generating transactions, but that's done now, so actualy ready for review now.

**Comment by justinvdm** [comment]
Also forgot about needing to fix the transaction serializer tests. Should be fixed now, all billing tests passing locally.

File: go/billing/models.py [hunk 1]
```diff
@@ -174,6 +187,12 @@ def message_credit_cost(self):
             self.message_cost, self.markup_percent)
 
     @property
+    def storage_credit_cost(self):
+        """Return the calculated cost per session (in credits)."""
+        return self.calculate_storage_credit_cost(
+            self.session_cost, self.markup_percent)
```
Comment (hodgestar) [inline go/billing/models.py]: `session_cost` -> `storage_cost`?

**Comment by hodgestar** [comment]
Other than the one question, looks good.

**Comment by justinvdm** [comment]
Comment resolved I think, should be ready for re-review.

**Comment by hodgestar** [comment]
:+1:

Let's figure out how to land and deploy on Monday.

**Comment by justinvdm** [comment]
#1118 includes this PR's changes, closing.

**Pull request closed without merging**